prow/config.yaml

```diff
@@ -27,13 +27,13 @@ branch-protection:
     kubernetes:
       repos:
         kubernetes/test-infra:
-          protect: true
-          # Contexts: inherited from presubmits
-          # Pushers: only admins
+          protect-by-default: true
+          require-contexts:
+          - cla/linuxfoundation
     kubernetes-sig-testing:
       repos:
-        frameworks:
-          protect: true
+        kubernetes-sig-testing/frameworks:
```

Comment:
Repo names shouldn't be prefixed by the org in my opinion. The org has already been specified based on which `Org` struct the `Repo` struct is nested under so the 'org/' prefix is redundant. It also seems confusing to require 'org/repo' format in a field called `repo`.

Reply:
Done